conans/client/tools/oss.py

```diff
@@ -558,6 +560,10 @@ def check_output(cmd, folder=None, return_code=False, stderr=None):
             raise CalledProcessErrorWithStderr(process.returncode, cmd, output=stderr)
 
         output = load(tmp_file)
+        try:
+            logger.info("Output: in file:{}\nstdout: {}\nstderr:{}".format(output, stdout, stderr))
+        except Exception as exc:
+            logger.info("Error logging command output: {}".format(exc))
```

Comment:
why not `logger.error` since it's an error?

Reply:
It is an error logging, not an error "error", but ok.